inc/endpoints/class-cache-endpoint.php

```diff
@@ -32,8 +32,9 @@ public function register_rest_routes() {
 			self::get_namespace(),
 			'/purge-cache',
 			[
-				'methods'  => \WP_REST_Server::CREATABLE,
-				'callback' => [ $this, 'get_route_response' ],
+				'callback'            => [ $this, 'get_route_response' ],
+				'methods'             => \WP_REST_Server::CREATABLE,
+				'permission_callback' => '__return_true',
```

Comment:
We should check for a capability in order to allow someone to clear the cache here. Maybe `current_user_can( 'manage_options' )` unless we want to give this capability to non-admins.

Otherwise, anyone in the world could send a POST request to an Irving endpoint and cause a cache stampede or construct a DoS attack.